### ISSUE TITLE
Frequentist RM ANOVA: Can't handle an object of class  “afex_aov” 

### ISSUE THREAD

**Issue by maciekszul** [jasp-stats/jasp-desktop#2412]
Trying to do basic repeated measures ANOVA (frequentist), I get persistent problem, nonexistent when doing bayesian version. I use 2x2 design. Problem occurs even with different datafile, with the same dimensions.
datafile if required: https://mfr.osf.io/render?url=https%3A%2F%2Fosf.io%2Fp54d3%2Fdownload

> Error in ref.grid(object = structure(list(anova_table = structure(list(: Can't handle an object of class  “afex_aov” 
> Use help('models', package = 'lsmeans') for information on supported models.
> 
> Stack trace
> .referenceGrid(options, fullModel)
> 
> lsmeans::lsmeans(fullModel, formula)
> 
> lsmeans.formula(fullModel, formula)
> 
> lsmeans(object, .all.vars(specs), by = by, ...)
> 
> lsmeans.character(object, .all.vars(specs), by = by, ...)
> 
> lsmeans.character.default(object, .all.vars(specs), by = by, ...)
> 
> lsmeans.default(object, specs, ...)
> 
> do.call('ref.grid', rgargs)
> 
> ref.grid(object = list(anova_table = list(`num Df` = c(1, 1, 1), `den Df` = c(20, 20, 20), MSE = c(0.0108026413700208, 0.00347051711256845, 0.000732690559138553), F = c(0.0833405077745035, 292.708562291153, 1.54468907258066), ges = c(0.000514463193833267, 0.367406072632858, 0.000646654695436145), `Pr(>F)` = c(0.775791372315324, 2.07928260353455e-13, 0.228295544969146)), aov = list(`(Intercept)` = list(coefficients = 0.643998015870239, residuals = 0, effects = -5.90233931025348, rank = 1, fitted.values = -5.90233931025348, 
>     assign = 0, qr = list(qr = -9.16515138991167, qraux = 9.16515138991167, pivot = 1, tol = 1e-07, rank = 1), df.residual = 0, terms = dependent ~ XbW9kYWxpdHk + XZGlmZmljdWx0eQ + XbW9kYWxpdHk:XZGlmZmljdWx0eQ), subject = list(coefficients = numeric(0), residuals = c(-0.374177338401451, 0.131830177142564, 0.115770757098317, 0.0468176732900825, 0.488956491402993, -0.0593103501108183, 0.0567930138863455, 0.102137646216566, -0.335552796216772, 0.352979796042815, 0.252412601135762, -0.242193832582156, 
> 0.308776668516115, 0.00668501009922841, 0.363293906631517, 0.276579036995121, -0.244385483805209, 0.194665705378545, -0.507752411549112, 0.0293786584346693), fitted.values = c(0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0), weights = NULL, rank = 0, df.residual = 20), `subject:XbW9kYWxpdHk` = list(coefficients = -0.00327380952023807, residuals = c(-0.244047619040476, -0.110714285740476, -0.198660714259524, -0.00595238090952437, -0.178869047659523, -0.0767857143095239, -0.085119047609524, 
> -0.0596726190404761, 0.0315476190404761, 0.0502976190404761, 0.126339285740476, -0.00907738090952386, 0.0534226190404761, 0.0190476190404762, 0.0180059523904761, 0.0784226190404762, -0.100744047659524, -0.00178571430952384, -0.110119047609524, -0.0580357143095238, 0.0336309523904761), effects = c(-0.0300049598747161, -0.0669983323211634, -0.242376667678837, -0.0496683343288369, -0.222585001078836, -0.120501667728836, -0.128835001028837, -0.0159566656211635, -0.0121683343788364, 0.00658166562116358, 
> 0.0826233323211636, -0.0527933343288364, 0.00970666562116355, -0.0246683343788363, -0.0257100010288365, 0.0347066656211637, -0.144460001078836, -0.0455016677288364, -0.153835001028836, -0.101751667728836, -0.0100850010288364), rank = 1, fitted.values = c(0.00654761904047621, 0.00654761904047617, -0.00654761904047615, -0.00654761904047615, -0.00654761904047615, -0.00654761904047615, -0.00654761904047614, 0.00654761904047615, -0.00654761904047615, -0.00654761904047615, -0.00654761904047615, -0.00654761904047615, 
> -0.00654761904047615, -0.00654761904047615, -0.00654761904047615, -0.00654761904047615, -0.00654761904047614, -0.00654761904047615, -0.00654761904047614, -0.00654761904047615, -0.00654761904047615), assign = 1, qr = list(qr = c(9.16515138991169, 0.218217890235992, -0.218217890235992, -0.218217890235992, -0.218217890235992, -0.218217890235992, -0.218217890235992, 0.218217890235992, -0.218217890235992, -0.218217890235992, -0.218217890235992, -0.218217890235992, -0.218217890235992, -0.218217890235992, 
> -0.218217890235992, -0.218217890235992, -0.218217890235992, -0.218217890235992, -0.218217890235992, -0.218217890235992, -0.218217890235992), qraux = 1.21821789023599, pivot = 1, tol = 1e-07, rank = 1), df.residual = 20, terms = dependent ~ XbW9kYWxpdHk + XZGlmZmljdWx0eQ + XbW9kYWxpdHk:XZGlmZmljdWx0eQ), `subject:XZGlmZmljdWx0eQ` = list(coefficients = 0.109970238096429, residuals = c(0.0053571427928558, 0.0970238094928572, -0.0168154761928571, 0.0758928571571425, -0.0175595238071427, 0.0407738095428573, 
> 0.0491071428428573, -0.0293154761928573, -0.105059523807143, 0.0324404761928574, 0.0480654761928571, -0.0623511904571429, 0.0168154761928574, 0.0675595238071429, -0.0644345237571428, -0.0147321428928571, -0.025148809492857, -0.0178571428428572, 0.0633928571571429, -0.115773809542857, 0.0529761904571429), effects = c(1.0078938805384, 0.0960641910146526, -0.0158558577146525, 0.0768524756353472, -0.0185191422853474, 0.0398141910646526, 0.0481475243646526, -0.0283558577146527, -0.106019142285347, 0.0314808577146527, 
> 0.0471058577146525, -0.0633108089353475, 0.0158558577146527, 0.0685191422853476, -0.0653941422353474, -0.0137725244146524, -0.0241891910146524, -0.0168975243646526, 0.0643524756353475, -0.114814191064653, 0.0539358089353476), rank = 1, fitted.values = c(-0.219940476192857, -0.219940476192857, 0.219940476192857, 0.219940476192857, -0.219940476192857, -0.219940476192857, -0.219940476192857, 0.219940476192857, -0.219940476192857, -0.219940476192857, -0.219940476192857, -0.219940476192857, -0.219940476192857, 
> 0.219940476192857, -0.219940476192857, 0.219940476192857, 0.219940476192857, 0.219940476192857, 0.219940476192857, 0.219940476192857, 0.219940476192857), assign = 2, qr = list(qr = c(9.16515138991168, 0.218217890235992, -0.218217890235992, -0.218217890235992, 0.218217890235992, 0.218217890235992, 0.218217890235992, -0.218217890235992, 0.218217890235992, 0.218217890235992, 0.218217890235992, 0.218217890235992, 0.218217890235992, -0.218217890235992, 0.218217890235992, -0.218217890235992, -0.218217890235992, 
> -0.218217890235992, -0.218217890235992, -0.218217890235992, -0.218217890235992), qraux = 1.21821789023599, pivot = 1, tol = 1e-07, rank = 1), df.residual = 20, terms = dependent ~ XbW9kYWxpdHk + XZGlmZmljdWx0eQ + XbW9kYWxpdHk:XZGlmZmljdWx0eQ), `subject:XbW9kYWxpdHk:XZGlmZmljdWx0eQ` = list(coefficients = 0.00367063492023816, residuals = c(-0.067658730159525, -0.0218253968595235, -0.0083829365404746, -0.0365079364904762, 0.0365079365404766, -0.0259920635095238, -0.00109126989047627, -0.00629960314047623, 
> -0.00307539685952351, -0.0176587301595237, -0.0229662698404763, 0.0125496031904765, 0.0228670634595236, -0.00307539685952377, -0.00620039680952362, -0.00203373015952368, -0.00828373015952369, -0.0301587302095237, 0.00109126989047629, -0.0344246031904763, -0.0490079364904763), effects = c(0.0336419247410791, -0.00970577074846901, -0.0205025626515291, -0.0486275626015307, 0.0486275626515311, -0.0138724373984693, -0.0132108960015308, -0.0184192292515307, 0.00904422925153098, -0.00553910404846919, -0.0350858959515307, 
> 0.024669229301531, 0.0107474373484691, 0.00904422925153074, 0.00591922930153089, 0.0100858959515308, 0.0038358959515308, -0.0180391040984692, 0.0132108960015308, -0.0465442293015308, -0.0611275626015308), rank = 1, fitted.values = c(-0.00734126984047631, -0.00734126984047633, 0.00734126984047631, 0.00734126984047632, -0.00734126984047632, -0.00734126984047632, 0.00734126984047632, 0.00734126984047631, -0.00734126984047631, -0.00734126984047631, 0.00734126984047631, -0.00734126984047632, 0.00734126984047632, 
> -0.00734126984047632, -0.00734126984047632, -0.00734126984047631, -0.00734126984047631, -0.00734126984047631, -0.00734126984047631, 0.00734126984047631, 0.00734126984047631), assign = 3, qr = list(qr = c(9.16515138991168, 0.218217890235993, -0.218217890235992, -0.218217890235993, 0.218217890235992, 0.218217890235993, -0.218217890235992, -0.218217890235992, 0.218217890235992, 0.218217890235992, -0.218217890235992, 0.218217890235992, -0.218217890235992, 0.218217890235993, 0.218217890235992, 0.218217890235992, 
> 0.218217890235992, 0.218217890235992, 0.218217890235992, -0.218217890235992, -0.218217890235992), qraux = 1.21821789023599, pivot = 1, tol = 1e-07, rank = 1), df.residual = 20, terms = dependent ~ XbW9kYWxpdHk + XZGlmZmljdWx0eQ + XbW9kYWxpdHk:XZGlmZmljdWx0eQ)), Anova = list(SSP = list(`(Intercept)` = 139.350437333454, XbW9kYWxpdHk = 0.00360119046833357, XZGlmZmljdWx0eQ = 4.06340029770702, `XbW9kYWxpdHk:XZGlmZmljdWx0eQ` = 0.00452711640113734), SSPE = list(`(Intercept)` = 5.79583002669577, XbW9kYWxpdHk = 0.864211309601665, 
>     XZGlmZmljdWx0eQ = 0.277641369005476, `XbW9kYWxpdHk:XZGlmZmljdWx0eQ` = 0.0586152447310843), P = list(`(Intercept)` = c(1, 1, 1, 1), XbW9kYWxpdHk = c(-1, -1, 1, 1), XZGlmZmljdWx0eQ = c(-1, 1, -1, 1), `XbW9kYWxpdHk:XZGlmZmljdWx0eQ` = c(1, -1, -1, 1)), df = c(1, 1, 1, 1), error.df = 20, terms = c('(Intercept)', 'XbW9kYWxpdHk', 'XZGlmZmljdWx0eQ', 'XbW9kYWxpdHk:XZGlmZmljdWx0eQ'), repeated = TRUE, type = 'III', test = 'Pillai', idata = list(XbW9kYWxpdHk = c(2, 2, 1, 1), XZGlmZmljdWx0eQ = c(2, 1, 2, 
> 1)), idesign = ~XbW9kYWxpdHk * XZGlmZmljdWx0eQ, icontrasts = c('contr.sum', 'contr.poly'), imatrix = NULL, singular = FALSE), lm = list(coefficients = c(0.754365079366667, 0.527083333333333, 0.753571428566667, 0.540972222214286), residuals = c(0.168551587333333, -0.231448412666667, -0.148115079366667, -0.00644841266666666, -0.146031746066667, 0.153968253933333, -0.123115079366667, 0.0143849206333334, 0.0998015873333333, -0.177281746066667, 0.176884920633333, 0.133134920633333, -0.104365079366667, 
> 0.187301587333333, 0.0414682539333333, 0.214384920633333, 0.0893849206333333, -0.106448412666667, 0.0831349206333334, -0.352281746066667, 0.0331349206333333, 0.106249999966667, -0.156250000033333, -0.122916666633333, -0.0458333333333334, -0.127083333333333, 0.168749999966667, -0.0729166666333334, 0.0499999999666666, -0.00833333333333332, -0.162500000033333, 0.247916666666667, 0.0833333333666667, -0.110416666633333, 0.116666666666667, -0.0291666666333334, 0.227083333366667, 0.106249999966667, -0.118750000033333, 
> 0.0208333333666667, -0.202083333333333, 0.0291666666666667, -0.143154761866665, -0.363988095266667, 0.0589285714333333, 0.0360119047333332, 0.0693452381333332, 0.204761904733333, -0.0369047618666667, -0.0389880952666667, 0.0651785714333332, -0.245238095266667, 0.0735119047333332, 0.154761904733333, -0.180654761866667, 0.165178571433333, 0.0172619047333333, 0.133928571433333, 0.181845238133333, -0.134821428566667, 0.194345238133333, -0.259821428566667, 0.0485119047333333, -0.0701388889142852, -0.245138888914286, 
> 0.0673611110857142, -0.0763888889142857, 0.0152777777857143, 0.271527777785714, 0.0111111110857143, -0.0159722222142857, -0.0368055555142857, -0.195138888914286, 0.0986111110857142, 0.0798611110857143, -0.140972222214286, 0.100694444485714, -0.0409722222142857, 0.150694444485714, 0.215277777785714, -0.0868055555142857, 0.129861111085714, -0.178472222214286, -0.0534722222142857), effects = c(-3.45693507782912, -0.261640853672998, -0.178307520372998, -0.036640853672998, -0.176224187072998, 0.123775812927002, 
> -0.153307520372998, -0.0158075203729979, 0.069609146327002, -0.207474187072998, 0.146692479627002, 0.102942479627002, -0.134557520372998, 0.157109146327002, 0.011275812927002, 0.184192479627002, 0.059192479627002, -0.136640853672998, 0.052942479627002, -0.382474187072998, 0.002942479627002, -2.41539927254964, -0.175282433406815, -0.141949100006815, -0.0648657667068154, -0.146115766706815, 0.149717566593185, -0.0919491000068154, 0.0309675665931846, -0.0273657667068153, -0.181532433406815, 0.228884233293185, 
> 0.0643008999931847, -0.129449100006815, 0.0976342332931847, -0.0481991000068154, 0.208050899993185, 0.0872175665931846, -0.137782433406815, 0.00180089999318467, -0.221115766706815, 0.0101342332931846, -3.45329811296276, -0.338344956742631, 0.0845717099573685, 0.0616550432573685, 0.0949883766573685, 0.230405043257369, -0.0112616233426315, -0.0133449567426315, 0.0908217099573685, -0.219594956742632, 0.0991550432573685, 0.180405043257369, -0.155011623342632, 0.190821709957368, 0.0429050432573685, 0.159571709957368, 
> 0.207488376657369, -0.109178290042631, 0.219988376657368, -0.234178290042631, 0.0741550432573685, -2.47904615716544, -0.232574994979509, 0.0799250050204905, -0.0638249949795094, 0.0278416717204906, 0.284091671720491, 0.0236750050204906, -0.00340832827950943, -0.0242416615795095, -0.182574994979509, 0.11117500502049, 0.0924250050204906, -0.128408328279509, 0.113258338420491, -0.0284083282795095, 0.163258338420491, 0.227841671720491, -0.0742416615795095, 0.14242500502049, -0.165908328279509, -0.0409083282795095
> ), rank = 1, fitted.values = c(0.754365079366667, 0.754365079366667, 0.754365079366667, 0.754365079366667, 0.754365079366667, 0.754365079366667, 0.754365079366667, 0.754365079366667, 0.754365079366667, 0.754365079366667, 0.754365079366667, 0.754365079366667, 0.754365079366667, 0.754365079366667, 0.754365079366667, 0.754365079366667, 0.754365079366667, 0.754365079366667, 0.754365079366667, 0.754365079366667, 0.754365079366667, 0.527083333333333, 0.527083333333333, 0.527083333333333, 0.527083333333333, 
> 0.527083333333333, 0.527083333333333, 0.527083333333333, 0.527083333333333, 0.527083333333333, 0.527083333333333, 0.527083333333333, 0.527083333333333, 0.527083333333333, 0.527083333333333, 0.527083333333333, 0.527083333333333, 0.527083333333333, 0.527083333333333, 0.527083333333333, 0.527083333333333, 0.527083333333333, 0.753571428566665, 0.753571428566667, 0.753571428566667, 0.753571428566667, 0.753571428566667, 0.753571428566667, 0.753571428566667, 0.753571428566667, 0.753571428566667, 0.753571428566667, 
> 0.753571428566667, 0.753571428566667, 0.753571428566667, 0.753571428566667, 0.753571428566667, 0.753571428566667, 0.753571428566667, 0.753571428566667, 0.753571428566667, 0.753571428566667, 0.753571428566667, 0.540972222214285, 0.540972222214286, 0.540972222214286, 0.540972222214286, 0.540972222214286, 0.540972222214286, 0.540972222214286, 0.540972222214286, 0.540972222214286, 0.540972222214286, 0.540972222214286, 0.540972222214286, 0.540972222214286, 0.540972222214286, 0.540972222214286, 0.540972222214286, 
> 0.540972222214286, 0.540972222214286, 0.540972222214286, 0.540972222214286, 0.540972222214286), assign = 0, qr = list(qr = c(-4.58257569495584, 0.218217890235992, 0.218217890235992, 0.218217890235992, 0.218217890235992, 0.218217890235992, 0.218217890235992, 0.218217890235992, 0.218217890235992, 0.218217890235992, 0.218217890235992, 0.218217890235992, 0.218217890235992, 0.218217890235992, 0.218217890235992, 0.218217890235992, 0.218217890235992, 0.218217890235992, 0.218217890235992, 0.218217890235992, 
> 0.218217890235992), qraux = 1.21821789023599, pivot = 1, tol = 1e-07, rank = 1), df.residual = 20, call = lm(formula = cbind(Xam95c3RpY2s_XZWFzeQ, Xam95c3RpY2s_XaGFyZA, Xa2V5Ym9hcmQ_XZWFzeQ, Xa2V5Ym9hcmQ_XaGFyZA) ~ 1, data = list(subject = 1:21, Xam95c3RpY2s_XZWFzeQ = c(0.9229166667, 0.5229166667, 0.60625, 0.7479166667, 0.6083333333, 0.9083333333, 0.63125, 0.76875, 0.8541666667, 0.5770833333, 0.93125, 0.8875, 0.65, 0.9416666667, 0.7958333333, 0.96875, 0.84375, 0.6479166667, 0.8375, 0.4020833333, 
> 0.7875), Xam95c3RpY2s_XaGFyZA = c(0.6333333333, 0.3708333333, 0.4041666667, 0.48125, 0.4, 0.6958333333, 0.4541666667, 0.5770833333, 0.51875, 0.3645833333, 0.775, 0.6104166667, 0.4166666667, 0.64375, 0.4979166667, 0.7541666667, 0.6333333333, 0.4083333333, 0.5479166667, 0.325, 0.55625), Xa2V5Ym9hcmQ_XZWFzeQ = c(0.6104166667, 0.3895833333, 0.8125, 0.7895833333, 0.8229166667, 0.9583333333, 0.7166666667, 0.7145833333, 0.81875, 0.5083333333, 0.8270833333, 0.9083333333, 0.5729166667, 0.91875, 0.7708333333, 
> 0.8875, 0.9354166667, 0.61875, 0.9479166667, 0.49375, 0.8020833333), Xa2V5Ym9hcmQ_XaGFyZA = c(0.4708333333, 0.2958333333, 0.6083333333, 0.4645833333, 0.55625, 0.8125, 0.5520833333, 0.525, 0.5041666667, 0.3458333333, 0.6395833333, 0.6208333333, 0.4, 0.6416666667, 0.5, 0.6916666667, 0.75625, 0.4541666667, 0.6708333333, 0.3625, 0.4875))), terms = cbind(Xam95c3RpY2s_XZWFzeQ, Xam95c3RpY2s_XaGFyZA, Xa2V5Ym9hcmQ_XZWFzeQ, Xa2V5Ym9hcmQ_XaGFyZA) ~ 1, model = list(`cbind(Xam95c3RpY2s_XZWFzeQ, Xam95c3RpY2s_XaGFyZA, Xa2V5Ym9hcmQ_XZWFzeQ, Xa2V5Ym9hcmQ_XaGFyZA)` = c(0.9229166667, 
> 0.5229166667, 0.60625, 0.7479166667, 0.6083333333, 0.9083333333, 0.63125, 0.76875, 0.8541666667, 0.5770833333, 0.93125, 0.8875, 0.65, 0.9416666667, 0.7958333333, 0.96875, 0.84375, 0.6479166667, 0.8375, 0.4020833333, 0.7875, 0.6333333333, 0.3708333333, 0.4041666667, 0.48125, 0.4, 0.6958333333, 0.4541666667, 0.5770833333, 0.51875, 0.3645833333, 0.775, 0.6104166667, 0.4166666667, 0.64375, 0.4979166667, 0.7541666667, 0.6333333333, 0.4083333333, 0.5479166667, 0.325, 0.55625, 0.6104166667, 0.3895833333, 
> 0.8125, 0.7895833333, 0.8229166667, 0.9583333333, 0.7166666667, 0.7145833333, 0.81875, 0.5083333333, 0.8270833333, 0.9083333333, 0.5729166667, 0.91875, 0.7708333333, 0.8875, 0.9354166667, 0.61875, 0.9479166667, 0.49375, 0.8020833333, 0.4708333333, 0.2958333333, 0.6083333333, 0.4645833333, 0.55625, 0.8125, 0.5520833333, 0.525, 0.5041666667, 0.3458333333, 0.6395833333, 0.6208333333, 0.4, 0.6416666667, 0.5, 0.6916666667, 0.75625, 0.4541666667, 0.6708333333, 0.3625, 0.4875))), data = list(long = list(
>     subject = c(1, 1, 1, 1, 2, 2, 2, 2, 3, 3, 3, 3, 4, 4, 4, 4, 5, 5, 5, 5, 6, 6, 6, 6, 7, 7, 7, 7, 8, 8, 8, 8, 9, 9, 9, 9, 10, 10, 10, 10, 11, 11, 11, 11, 12, 12, 12, 12, 13, 13, 13, 13, 14, 14, 14, 14, 15, 15, 15, 15, 16, 16, 16, 16, 17, 17, 17, 17, 18, 18, 18, 18, 19, 19, 19, 19, 20, 20, 20, 20, 21, 21, 21, 21), XbW9kYWxpdHk = c(1, 1, 2, 2, 1, 1, 2, 2, 1, 1, 2, 2, 1, 1, 2, 2, 1, 1, 2, 2, 1, 1, 2, 2, 1, 1, 2, 2, 1, 1, 2, 2, 1, 1, 2, 2, 1, 1, 2, 2, 1, 1, 2, 2, 1, 1, 2, 2, 1, 1, 2, 2, 1, 1, 2, 2, 
>     1, 1, 2, 2, 1, 1, 2, 2, 1, 1, 2, 2, 1, 1, 2, 2, 1, 1, 2, 2, 1, 1, 2, 2, 1, 1, 2, 2), XZGlmZmljdWx0eQ = c(1, 2, 1, 2, 1, 2, 1, 2, 1, 2, 1, 2, 1, 2, 1, 2, 1, 2, 1, 2, 1, 2, 1, 2, 1, 2, 1, 2, 1, 2, 1, 2, 1, 2, 1, 2, 1, 2, 1, 2, 1, 2, 1, 2, 1, 2, 1, 2, 1, 2, 1, 2, 1, 2, 1, 2, 1, 2, 1, 2, 1, 2, 1, 2, 1, 2, 1, 2, 1, 2, 1, 2, 1, 2, 1, 2, 1, 2, 1, 2, 1, 2, 1, 2), dependent = c(0.9229166667, 0.6333333333, 0.6104166667, 0.4708333333, 0.5229166667, 0.3708333333, 0.3895833333, 0.2958333333, 0.60625, 0.4041666667, 
>     0.8125, 0.6083333333, 0.7479166667, 0.48125, 0.7895833333, 0.4645833333, 0.6083333333, 0.4, 0.8229166667, 0.55625, 0.9083333333, 0.6958333333, 0.9583333333, 0.8125, 0.63125, 0.4541666667, 0.7166666667, 0.5520833333, 0.76875, 0.5770833333, 0.7145833333, 0.525, 0.8541666667, 0.51875, 0.81875, 0.5041666667, 0.5770833333, 0.3645833333, 0.5083333333, 0.3458333333, 0.93125, 0.775, 0.8270833333, 0.6395833333, 0.8875, 0.6104166667, 0.9083333333, 0.6208333333, 0.65, 0.4166666667, 0.5729166667, 0.4, 0.9416666667, 
>     0.64375, 0.91875, 0.6416666667, 0.7958333333, 0.4979166667, 0.7708333333, 0.5, 0.96875, 0.7541666667, 0.8875, 0.6916666667, 0.84375, 0.6333333333, 0.9354166667, 0.75625, 0.6479166667, 0.4083333333, 0.61875, 0.4541666667, 0.8375, 0.5479166667, 0.9479166667, 0.6708333333, 0.4020833333, 0.325, 0.49375, 0.3625, 0.7875, 0.55625, 0.8020833333, 0.4875)), wide = list(subject = 1:21, Xam95c3RpY2s_XZWFzeQ = c(0.9229166667, 0.5229166667, 0.60625, 0.7479166667, 0.6083333333, 0.9083333333, 0.63125, 0.76875, 
> 0.8541666667, 0.5770833333, 0.93125, 0.8875, 0.65, 0.9416666667, 0.7958333333, 0.96875, 0.84375, 0.6479166667, 0.8375, 0.4020833333, 0.7875), Xam95c3RpY2s_XaGFyZA = c(0.6333333333, 0.3708333333, 0.4041666667, 0.48125, 0.4, 0.6958333333, 0.4541666667, 0.5770833333, 0.51875, 0.3645833333, 0.775, 0.6104166667, 0.4166666667, 0.64375, 0.4979166667, 0.7541666667, 0.6333333333, 0.4083333333, 0.5479166667, 0.325, 0.55625), Xa2V5Ym9hcmQ_XZWFzeQ = c(0.6104166667, 0.3895833333, 0.8125, 0.7895833333, 0.8229166667, 
> 0.9583333333, 0.7166666667, 0.7145833333, 0.81875, 0.5083333333, 0.8270833333, 0.9083333333, 0.5729166667, 0.91875, 0.7708333333, 0.8875, 0.9354166667, 0.61875, 0.9479166667, 0.49375, 0.8020833333), Xa2V5Ym9hcmQ_XaGFyZA = c(0.4708333333, 0.2958333333, 0.6083333333, 0.4645833333, 0.55625, 0.8125, 0.5520833333, 0.525, 0.5041666667, 0.3458333333, 0.6395833333, 0.6208333333, 0.4, 0.6416666667, 0.5, 0.6916666667, 0.75625, 0.4541666667, 0.6708333333, 0.3625, 0.4875)), idata = list(XbW9kYWxpdHk = c(2, 2, 
> 1, 1), XZGlmZmljdWx0eQ = c(2, 1, 2, 1)))), by = NULL)
> 
> stop(data)

* JASP version: Version 0.8.6 (flatpak install)
* OS name and version: Ubuntu 16.04.4 LTS 64-bit
* Analysis: Repeated Measures ANOVA

**Comment by maciekszul** [comment]
Issue nonexistent when JASP is installed via PPA on the same machine, using the same data and same analysis.

**Comment by JohnnyDoorn** [comment]
Hi Maciekszul,
Thanks for your report! I ran into a similar problem on the flatpak distribution. It has to do with the R-package lsmeans being deprecated. Becaues the flatpak distribution has its own R-packages (for lsmeans, the latest version), it behaves differently from the ppa version, which uses the locally installed packages. If you were to update all your packages your ppa version would also not run RM ANOVA's. I updated the code in JASP, so now it uses the new version of lsmeans, called emmeans. So it should be fixed in the next release =)
Kind regards,
Johnny
